packages/clerk-js/src/ui/components/OrganizationProfile/AddDomainForm.tsx

```diff
@@ -53,7 +54,7 @@ export const AddDomainForm = withCardStateProvider((props: AddDomainFormProps) =
         wizard.nextStep();
       })
       .catch(err => {
-        handleError(err, [nameField], card.setError);
+        handleError(err, [nameField], err => card.setError(translateError(err)));
```

Comment:
I'd expect this to work as it was, are you seeing something different ?
I believe the pattern here it to use <Card.Alert/> for global errors which handles the localization

Reply:
Thanks for catching that! I didn't recognize that `Card.Alert` already handles localization. 

I've updated my changes and only using `translateError` directly on `InviteMembersForm` to concatenate the `email` value on `already_a_member_in_organization` error code.